src/modules/common/containers/__tests__/Error.js

```diff
@@ -3,30 +3,35 @@ import { Provider } from 'react-redux'
 import renderer from 'react-test-renderer'
 import { mount } from 'enzyme'
 import Error from '../Error'
-import mockStore from '__mocks__/store'
+import configureMockStore from 'redux-mock-store'
+import { routerForBrowser } from 'redux-little-router'
 
 jest.mock('react-i18next')
 
 describe('Error', () => {
+  const router = routerForBrowser({routes: {}})
+  const mockStore = configureMockStore([router.middleware])
+
   test('should match snapshot', () => {
     const component = renderer.create(
-      <Provider store={mockStore}><Error error="Error Message"/></Provider>
+      <Provider store={mockStore()}><Error error="Error Message"/></Provider>
     )
 
     expect(component.toJSON()).toMatchSnapshot()
```

Comment:
toJSON is not needed

Reply:
Will be done in the task for common module.